src/smart-components/group/principal/principals.js

```diff
@@ -74,7 +75,7 @@ const GroupPrincipals = () => {
   };
 
   const actionResolver = () =>
-    !(userIdentity && userIdentity.user && userIdentity.user.is_org_admin)
+    !(orgAdmin || userAccessAdministrator)
```

Comment:
Can you consolidate the condition into a variable so we don't evaluate it all the time?